modules/lvms-uninstalling-openshift-data-foundation-logical-volume-manager-operator.adoc

```diff
@@ -0,0 +1,311 @@
+// Module included in the following assemblies:
+//
+// storage/logical-volume-manager-storage.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-uninstalling-lvms-rhacm_{context}"]
+= Uninstalling logical volume manager storage installed using {rh-rhacm}
+
+To uninstall the logical volume manager storage when you installed the Operator using {rh-rhacm}, you need to delete the {rh-rhacm} policy that you created for deploying and configuring the Operator.
+However, when you delete the {rh-rhacm} policy, the resources that the policy has created are not removed.
+You need to create additional policies to remove the resources.
+
+As the created resources are not removed when you delete the policy, you need to perform the following steps:
+
+* Remove all the PVCs and volume snapshots provisioned by the logical volume manager storage.
+* Remove the `LVMCluster` resources to clean up the Logical Volume Manager resources created on the disks.
+* Create an additional policy to uninstall the Operator.
+
+.Prerequisites
+
+* Ensure that the following are deleted before deleting the policy:
+** All the applications on the managed clusters that are using the storage provisioned by the logical volume manager storage.
+** Persistent volume claims (PVCs) and persistent volumes (PVs) provisioned using the logical volume manager storage.
+** All volume snapshots provisioned by the logical volume manager storage.
+* Access to the {rh-rhacm} cluster using an account with a `cluster-admin` role.
+
+.Procedure
+
+. In the OpenShift command-line interface, delete the ACM policy that you created for deploying and configuring the logical volume manager storage on the hub cluster by using the following command:
++
+[source,terminal]
+----
+# oc delete -f policy-lvms-operator.yaml -n lvms-policy-ns
```

Comment:
It's an example, but one that the docs use since the installation (and that's where it is made clear that it's an example): https://55472--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.html#lvms-installing-odf-logical-volume-manager-operator-using-rhacm_logical-volume-manager-storage

Would you still recommend using `<>`?